config.base.yaml

```diff
@@ -0,0 +1,18 @@
+stores:
+  filestore:
+    plugin: cloudmarker.stores.filestore.FileStore
+    params:
+      path: '/tmp/cloudmarker'
+
+clouds:
+  mockcloud:
+    plugin: cloudmarker.clouds.mockcloud.MockCloud
+
+mockjob:
+  clouds:
+    - mockcloud
+  stores:
+    - filestore
+
```

Comment:
In this structure, is the stores config like a default? Would the clouds also in future support specifying their own specific stores?